### DISCUSSION TITLE
Apology for Recent v2.5.1 Release Issue

### DISCUSSION THREAD

**Discussion by abraunegg** [abraunegg/onedrive#2861] [General]
I sincerely apologise for the recent update that introduced a significant bug, which unfortunately broke client usage for those who upgraded. I understand how frustrating this was and deeply regret the inconvenience it caused.

The issue has now been fixed, and the updated version is available. I’m reviewing my processes to hopefully prevent this sort of issue from happening again in the future.

Thank you for your patience and ongoing support.

**Comment by Weissrolf** [comment]
Thanks for the update. Unfortunately the installation via OpenSuSE Build Service repository still fetches the broken 2.5.1 instead of the fixed 2.5.2.

**Reply by abraunegg** [reply]
I understand your concern, but this is currently beyond my control. The issue should be resolved within the next 12 hours or so, as @norbusan is enjoying their weekend at the moment.

If you're looking for a quicker or more permanent solution, you might consider advocating for your preferred distribution to offer more up-to-date packages and a more efficient way to manage updates. Alternatively, you could switch to a distribution like Arch, Fedora, or Solus, which typically provide more up-to-date packages, without relying on external repositories like the OpenSuSE Repository for Debian/Ubuntu-based systems.

**Reply by Weissrolf** [reply]
I will just wait, thanks for the quick reply. VMs on Oracle usually run on Ubuntu and I followed your instruction on how to set up Onedrive via Suse repository. This works well and I am eager to see how the new multi-thread changes work with the many small files I often sync (upload only so that my NAS can back them up externally from Oracle servers).

**Reply by norbusan** [reply]
As Alex said, I'm out traveling and will return in a few hours, after which I will upload new packages.

**Reply by norbusan** [reply]
@Weissrolf I just sent the files off to OBS, should be ready in rather short time (depending on your OS and arch).

**Comment by Weissrolf** [comment]
They are live now. Thanks again and have a nice week!

**Comment by Langtude-Larson** [comment]
Thank you guys a lot for the quick fix! appreciate it!
